### PR TITLE
Sensors Feed added

### PR DESCRIPTION
**Pull request by WRIOLtd**

### CODE REVIEW

File: testbed/sensorID-1bc6c4e6-63b2-433b-90f7-db1910e05dc0/feed/index.html [hunk 1]
```diff
@@ -9,24 +10,24 @@
...
-        }
-        ]
+            "@context": "https://schema.org",
+            "@type": "ItemList",
+            "name": "Feed",
+            "itemListElement": [{
+                "@type": "ItemList",
+                "name": "Feed",
+                "description": "Feed for Demo",
+                "url": "https://wriodev.github.io/sensorID-a575d13e-b559-4d19-bb2a-1a3cd67952e7/"
```
Comment (AlexeyAnshakov) [inline testbed/sensorID-1bc6c4e6-63b2-433b-90f7-db1910e05dc0/feed/index.html]: only ID, delete
"sensorID-"

Comment (AlexeyAnshakov) [reply testbed/sensorID-1bc6c4e6-63b2-433b-90f7-db1910e05dc0/feed/index.html]: Move to imec.wr.io

File: testbed/sensorID-1bc6c4e6-63b2-433b-90f7-db1910e05dc0/index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Demo">
+    <meta name="author" content="WRIO Ltd.">
+    <meta name="keywords" content="IMEC, Demo, WRIO, hub">
+    <title>ID_here &middot; Sensor</title>
```
Comment (AlexeyAnshakov) [inline testbed/sensorID-1bc6c4e6-63b2-433b-90f7-db1910e05dc0/index.html]: Sensor ID in the title

File: testbed/sensorID-1bc6c4e6-63b2-433b-90f7-db1910e05dc0/index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    <script type="application/ld+json">
+        {
+            "@context": "https://schema.org",
+            "@type": "ItemList",
+            "name": "Feed",
+            "itemListElement": [{
+                "@type": "ItemList",
+                "name": "Feed item",
+                "description": "Demo feed",
+                "url": "https://imec.wr.io/testbed/sensorID-1bc6c4e6-63b2-433b-90f7-db1910e05dc0/feed/?feed"
```
Comment (AlexeyAnshakov) [inline testbed/sensorID-1bc6c4e6-63b2-433b-90f7-db1910e05dc0/index.html]: Delete
sensorID-

File: testbed/sensorID-95947812-85b3-4d81-92a4-fca29790f72c/feed/index.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    <script type="application/ld+json">
+        {
+            "@context": "https://schema.org",
+            "@type": "ItemList",
+            "name": "Feed",
+            "itemListElement": [{
+                "@type": "ItemList",
+                "name": "Feed",
+                "description": "Feed for Demo",
+                "url": "https://wriodev.github.io/sensorID-a575d13e-b559-4d19-bb2a-1a3cd67952e7/"
```
Comment (AlexeyAnshakov) [inline testbed/sensorID-95947812-85b3-4d81-92a4-fca29790f72c/feed/index.html]: Move to imec.wr.io
The same for all pages

File: testbed/sensorID-a575d13e-b559-4d19-bb2a-1a3cd67952e7/index.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    <script type="application/ld+json">
+        {
+            "@context": "https://schema.org",
+            "@type": "Article",
+            "inLanguage": "En",
+            "keywords": "IMEC, Demo, WRIO, hub",
+            "author": "WRIO Ltd.",
+            "name": "About IMEC",
+            "thumbnailUrl": "https://imec.wr.io/img/cover1-200x200.jpg",
+            "about": "IMEC sensor ID_here.",
```
Comment (AlexeyAnshakov) [inline testbed/sensorID-a575d13e-b559-4d19-bb2a-1a3cd67952e7/index.html]: Add ID

**Review by AlexeyAnshakov — CHANGES_REQUESTED** [review]

File: testbed/sensorID-1bc6c4e6-63b2-433b-90f7-db1910e05dc0/feed/index.html (hunk 1 above)
Comment (WRIOLtd) [reply testbed/sensorID-1bc6c4e6-63b2-433b-90f7-db1910e05dc0/feed/index.html]: updated!

**Review by AlexeyAnshakov — CHANGES_REQUESTED** [review]
See comments

File: testbed/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed/index.html [hunk 6]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (AlexeyAnshakov) [inline testbed/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed/index.html:1]: This file must contains feed data. The same for other sensor feed pages

Comment (WRIOLtd) [reply testbed/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed/index.html:1]: I'm loading feed data through url!,
isnt it right ?
ttps://imec.wr.io/a575d13e-b559-4d19-bb2a-1a3cd67952e7/

Comment (AlexeyAnshakov) [reply testbed/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed/index.html:1]: No. see issue
https://github.com/webRunes/WRIO-InternetOS/issues/1610
ttps://imec.wr.io/a575d13e-b559-4d19-bb2a-1a3cd67952e7/ is a sensor page
ttps://imec.wr.io/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed is a feed page (sensor's data) of the sensor

Comment (AlexeyAnshakov) [reply testbed/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed/index.html:1]: https://github.com/WRIO-Hubs/imec.wr.io/ already contains example pages.

Comment (WRIOLtd) [reply testbed/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed/index.html:1]: Sorry, I was talking about 
ttps://imec.wr.io/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed
here I am getting data through url.
or simply I should add all data in this file ?
like this = https://raw.githubusercontent.com/WRIOdev/sensorID-a575d13e-b559-4d19-bb2a-1a3cd67952e7/gh-pages/index.html

Comment (AlexeyAnshakov) [reply testbed/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed/index.html:1]: Yes, add feed data.

Comment (WRIOLtd) [reply testbed/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed/index.html:1]: Ok

Comment (WRIOdev) [reply testbed/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed/index.html]: what is the pupose of adding the url here, when I will be adding feed data directly ?

**Pull request merged**